spinalcordtoolbox/moco.py

```diff
@@ -867,6 +867,9 @@ def moco_wrapper_interleaved(param):
     orig_name = param.fname_data    # store orig name passed by user
     file_data = 'data.nii'          # corresponds to the full input data (e.g. dmri or fmri)
     file_mask = 'mask.nii'
+    file_moco_params_x = 'moco_params_x.nii.gz'
+    file_moco_params_y = 'moco_params_y.nii.gz'
+    file_moco_params_csv = 'moco_params.tsv'
```

Comment:
weird to call variable with _csv suffix if extension is tsv

Reply:
You are right. I chose this variable name based on  https://github.com/neuropoly/spinalcordtoolbox/blob/be4f8e9890bfc0ba141a8961d21bc8a12c5c979b/spinalcordtoolbox/moco.py#L131
When you prefer csv over tsv, should I switch to csv over whole moco script?

Reply:
yes, csv would be nice.

Reply:
Actually no, let’s keep tsv. Let’s not extend the purpose of this PR too much.

Reply:
okay, I preserved tsv and added future TODO to replace tsv by csv